src/renderer/imgproxy.ts

```diff
@@ -0,0 +1,72 @@
+import { GenericStorageBackend } from '../backend/generic'
+import axios, { Axios } from 'axios'
+
+interface TransformOptions {
+  width?: number
+  height?: number
+  resize?: 'fill' | 'fit' | 'fill-down' | 'force' | 'auto'
+}
+
+interface ImgProxyOptions {
+  url: string
+}
+
+const LIMITS = {
+  height: {
+    max: 5000,
```

Comment:
Lets make these configurable anyway, we might want to offer different limits to different users

Reply:
ok, for the MVP we'll have these values as envs